configs/.env

```diff
@@ -0,0 +1,19 @@
+# GITHUB_ACCESS_TOKEN=
```

Comment:
This is going to be used as a module, please make these params instead of load-env.